modules/base/src/io/datvolumewriter.cpp

```diff
@@ -93,24 +95,36 @@ void writeDatVolume(const Volume& data, const std::filesystem::path& filePath,
     glm::mat4 wtm = glm::transpose(data.getWorldMatrix());
 
     auto print = util::overloaded{
-        [&](std::string_view key, const std::string& val) { fmt::print(ss, "{}: {}\n", key, val); },
-        [&](std::string_view key, std::string_view val) { fmt::print(ss, "{}: {}\n", key, val); },
-        [&](std::string_view key, InterpolationType val) { fmt::print(ss, "{}: {}\n", key, val); },
-        [&](std::string_view key, const SwizzleMask& mask) {
+        [&ss](std::string_view key, const std::string& val) {
```

Comment:
Is `string&` needed when string_view is there?

Reply:
Doesn't seem to be the case anymore. Will remove.